[2024-05-20T03:26:57.405+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: api_data_attendance.create_att_data_into_db scheduled__2024-05-19T00:00:00+00:00 [queued]>
[2024-05-20T03:26:57.427+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: api_data_attendance.create_att_data_into_db scheduled__2024-05-19T00:00:00+00:00 [queued]>
[2024-05-20T03:26:57.429+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 4
[2024-05-20T03:26:57.452+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): create_att_data_into_db> on 2024-05-19 00:00:00+00:00
[2024-05-20T03:26:57.463+0000] {standard_task_runner.py:60} INFO - Started process 221 to run task
[2024-05-20T03:26:57.480+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'api_data_attendance', 'create_att_data_into_db', 'scheduled__2024-05-19T00:00:00+00:00', '--job-id', '596', '--raw', '--subdir', 'DAGS_FOLDER/api_data_attendance.py', '--cfg-path', '/tmp/tmpgvnr9g8n']
[2024-05-20T03:26:57.487+0000] {standard_task_runner.py:88} INFO - Job 596: Subtask create_att_data_into_db
[2024-05-20T03:26:57.627+0000] {task_command.py:423} INFO - Running <TaskInstance: api_data_attendance.create_att_data_into_db scheduled__2024-05-19T00:00:00+00:00 [running]> on host e3e2dc6bc2be
[2024-05-20T03:26:57.960+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='api_data_attendance' AIRFLOW_CTX_TASK_ID='create_att_data_into_db' AIRFLOW_CTX_EXECUTION_DATE='2024-05-19T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-05-19T00:00:00+00:00'
[2024-05-20T03:26:57.980+0000] {base.py:83} INFO - Using connection ID 'postgres_conn' for task execution.
[2024-05-20T03:26:58.000+0000] {base.py:83} INFO - Using connection ID 'postgres_conn' for task execution.
[2024-05-20T03:26:58.018+0000] {sql.py:450} INFO - Running statement: 
        CREATE TABLE IF NOT EXISTS attendance (
            id SERIAL PRIMARY KEY,
            subject_code VARCHAR(10),
            subject_name VARCHAR(255),
            section VARCHAR(10),
            num_enrolledStd INT,
            num_present INT,
            percent_present DECIMAL(5,2),
            num_absent INT,
            percent_absent DECIMAL(5,2),
            num_leave INT,
            percent_leave DECIMAL(5,2),
            class_date DATE,
            class_start TIME,
            class_end TIME
        );
        , parameters: None
[2024-05-20T03:26:58.028+0000] {python.py:201} INFO - Done. Returned value was: None
[2024-05-20T03:26:58.051+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=api_data_attendance, task_id=create_att_data_into_db, execution_date=20240519T000000, start_date=20240520T032657, end_date=20240520T032658
[2024-05-20T03:26:58.144+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-05-20T03:26:58.219+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
